semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/TypeOps.scala

```diff
@@ -71,6 +71,7 @@ trait TypeOps {
          */
         case g.WildcardType => s.NoType
         case _: g.BoundedWildcardType => s.NoType
+        case _: g.ClassInfoType => s.NoType
```

Comment:
If you don't have the time to add the test, do let me know!

Reply:
@tgodzik Sounds good! I should have some time to try to spec this later this afternoon.

Reply:
@tgodzik Maybe a stupid question, but I can't seem to actually run any of the test cases that use `targeted` from `SemanticdbSuite`, e.g.,
```
sbt:scalametaRoot> testsSemanticdb/testOnly *Issue2024Suite
[info] Passed: Total 0, Failed 0, Errors 0, Passed 0
```

It seems that none of the tests defined using `targeted` are excited :thinking:

Reply:
I added a first attempt at a test as part of [this commit](https://github.com/scalameta/scalameta/commit/03249846bc08a16b650bf74eee90e2665e79c74a) but simply pasting in `AnyRef.scala` fails to compile with
```
<input>:4:7: error illegal cyclic inheritance involving type AnyRef
```.

I could probably use some help here with this if you would have the time.

Reply:
You might need to change the name to not conflict with the standard AnyRef

As for the tests this might work:
`testsSemanticdb/testOnly -- *Issue2024Suite`

Or a full name of the suite

Reply:
> You might need to change the name to not conflict with the standard AnyRef

I think the whole point is that code exists which uses the standard `AnyRef`, and @masonedmison is trying to make sure semanticdb doesn't fail on it.

As to the error on cyclic inheritance: do a web search or ask an LLM what to do about it, they will likely help.

Is it possible that excluding the entire `ClassInfoType` is not the right thing to do, as it doesn't just pertain to this case but to all inheritance cases? Perhaps something more narrow?

Reply:
@kitbellew I just pushed up a commit that matches the `typeSymbol` of the `ClassInfoType` value. Please correct me if there is a better way to do this.